org.flightgear.FlightGear.yaml

```diff
@@ -15,11 +15,16 @@ add-extensions:
 command: flightgear.sh
 rename-icon: flightgear
 finish-args:
-- --socket=wayland
-- --socket=fallback-x11
+# shm + IPC are required for X11, otherwise the performances aren't great
 - --socket=x11
+- --device=shm
 - --share=ipc
+# Joysticks, yokes, etc. aren't handled by libinput/wayland/x11, so the application
+# needs direct access to those peripherals.
 - --device=all
+# Enable GPU acceleration (this is already covered by device=all but if one day
+# it gets removed, then we'll still have it).
+- --device=dri
```

Comment:
You already have device=dri with device=all.

Reply:
I know that, but it's so it's more clear for future versions (if one day we wouldn't need device=all for joysticks/yoke).

Reply:
Does it suit you if I let it there ?